src/Compilers/CSharp/Portable/Operations/CSharpOperationNodes.cs

```diff
@@ -858,17 +860,33 @@ internal sealed class CSharpLazyAnonymousFunctionOperation : LazyAnonymousFuncti
     {
         private readonly CSharpOperationFactory _operationFactory;
         private readonly BoundNode _body;
+        private readonly ImmutableArray<ConditionalOperation> _nullChecksToPrepend;
```

Comment:
>_nullChecksToPrepend [](start = 62, length = 20)

Consider using the same name for the field and constructor parameter here in all three classes. (I think I prefer `_prependedNullChecks`.)